consensus/src/common.rs

```diff
@@ -131,14 +132,20 @@ impl Chain<ProtocolState> for ProtocolStateChain {
         self.0.len()
     }
 
-    fn last_vrf_hash(&self) -> Option<Fp> {
-        static HASHER_POOL: OnceCell<SpinLockObjectPool<PoseidonHasherKimchi<VrfOutputTruncated>>> =
-            OnceCell::new();
-        let pool =
-            HASHER_POOL.get_or_init(|| SpinLockObjectPool::new(|| create_kimchi(()), |_| ()));
-        let mut hasher = pool.pull();
-        self.top()
-            .map(|s| hasher.hash(&s.body.consensus_state.last_vrf_output))
+    fn last_vrf_hash(&self) -> Result<String, ConsensusError> {
+        use blake2::digest::VariableOutput;
+        let mut hasher = Blake2bVar::new(32).unwrap();
```

Comment:
Any chance to avoid doing `.unwrap` here?

Reply:
Yeah, i have to refactor this function. Need to also move the constant out and the use statement above.